install-panel.sh

```diff
@@ -603,10 +603,21 @@ function debian_based_letsencrypt {
 
   systemctl stop nginx
 
-  echo -e "\nMake sure you choose Option 1, and create a Standalone Web Server during the certificate"
-  certbot certonly -d "$FQDN"
+  certbot certonly --standalone -d "$FQDN" || FAILED=true
```

Comment:
```suggestion
  FAILED=false
  certbot certonly --standalone -d "$FQDN" || FAILED=true
```
In reference to https://github.com/vilhelmprytz/pterodactyl-installer/pull/68#issuecomment-662536826

I think you are very much correct, this might work?